package.json

```diff
@@ -44,21 +51,24 @@
     "hapi-auth-cookie": "^2.0.0",
     "hoek": "^2.11.1",
     "hyperquest": "^1.0.1",
+    "imports-loader": "^0.6.3",
     "joi": "^6.0.8",
+    "jsx-loader": "^0.12.2",
     "react": "^0.12.2",
     "react-router": "^0.12.4",
+    "react-router-stub": "0.0.5",
     "react-validation-mixin": "^4.0.3",
+    "should": "^5.2.0",
```

Comment:
sadly not, since the build step *also* builds our JS files it's actually a `build:js` dependency